deploy/docker-compose.yaml

```
@@ -11,7 +11,7 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: admin
       POSTGRES_DB: users
-      TRIP_RESEARCHER_ADDRESS: localhost:8000
+      TRIPS_MS_ADDRESS: localhost:8000
```

Nie localhost tylko nazwa serwisu dla `trip ms`. 
```suggestion
      TRIPS_MS_ADDRESS: trips:8000
```